msteams-platform/tabs/how-to/app-caching.md

```diff
@@ -22,61 +24,83 @@ App caching is supported for the following:
 
-When you opt into app caching, the iframe or webview that is used to host the embedded app is reused as users navigate to different instances of the app within a window. The iframe or webview used to host the app is hidden when the users leave the app and shown when the users return to the app.
+When you opt into app suspension, the iframe or webview that is used to host the embedded app is reused as users navigate to different instances of the app within a window. The iframe or webview used to host the app is hidden when the users leave the app and shown when the users return to the app.
 
 > [!NOTE]
-> If the app caching isn't enabled, the iframe or webview is recreated every time the user launches the app.
+> If app suspension isn't enabled, the iframe or webview is recreated every time the user launches the app.
+
+There are multiple reasons for an app to not get suspended or for an app to get removed from the cache. Some general reasons across Microsoft 365 apps are:
+
+* The total memory load is high.
+* The total number of suspended apps exceed the maximum cache size. In this cirumcstance the oldest suspended app is removed.
+* The app is terminated if the machine's available memory is low.
+* The app is suspended for a long time without getting resumed.
 
-There are multiple reasons for an app to not get cached or for an app to get removed from the cache, some of the reasons are (numbers here are subject to change):
+Within the Teams app some of the reasons are (numbers here are subject to change):
 
 * If the system memory load is high, the app is removed from the cache.
-* If the number of cached apps exceeds the maximum cache size, the oldest cached app is removed from the cache.
-* The app isn't cached if Teams doesn't receive the `readyToUnload` signal from the app within 30 seconds after sending the `beforeUnload` notification.
-* App caching is disabled if the system memory is less than 4 GB or if the available memory is less than 1 GB on Windows or 512 MB on Mac.
-* Side panel is the only supported frameContext for app caching in meetings.
-* App caching isn't supported for meetings where the invited user count is more than 20.
-* If an app fails to load, the app isn't cached.
+* If the number of suspended apps exceeds the maximum cache size, the oldest suspended app is removed from the cache.
+* The app isn't suspended if Teams doesn't receive the `readyToUnload` signal from the app within 30 seconds after sending the `beforeUnload` notification.
+* App suspension is disabled if the system memory is less than 4 GB or if the available memory is less than 1 GB on Windows or 512 MB on Mac.
+* Side panel is the only supported frameContext for app suspension in meetings.
+* App suspesnion isn't supported for meetings where the invited user count is more than 20.
+* If an app fails to load, the app isn't suspended.
```

Comment:
If an app fails to load, the app is terminated

Reply:
Thanks, updated in upcoming commit.